.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -22,6 +22,7 @@ Please check followings before submitting a new issue.
 - [ ] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
 - [ ] I have already searched existing issues and they are not help  to me
 - [ ] I examined error or warning messages and it's difficult to solve
+- [ ] Using [the latest](https://www.npmjs.com/package/hexo?activeTab=versions) version of Hexo (run `hexo version` to check)
```

Comment:
Should we use `release` page at GitHub? The release page has the changelog while npm page doesn't.